### PR TITLE
Publish to npm

### PR DESCRIPTION
**Pull request by ehmicky**
See background at https://github.com/netlify/build/pull/2076#discussion_r532657394

This publish the `plugins.json` to npm under the package name `@netlify/plugins-list`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
-  "name": "netlify-plugins",
+  "name": "@netlify/plugins-list",
   "private": true,
   "version": "1.0.0",
-  "description": "Netlify plugins",
+  "description": "List of Netlify plugins",
+  "main": "site/plugins.json",
+  "files": [],
```
Comment (ehmicky) [inline package.json:7]: `main` is always included in `files` even when `files` is an empty array.
I double-checked with `npm publish --dry`.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -33,4 +33,7 @@ Netlify will try to reach the plugin author(s), and if that fails, we will add a
 
 ## Releasing
 
-See the [documentation in the buildbot](https://github.com/netlify/buildbot/blob/master/docs/overview.md#build-plugins).
+```
+$ npm version major|minor|patch
+$ npm publish
+```
```
Comment (ehmicky) [inline docs/CONTRIBUTING.md:39]: I will remove the documentation in the buildbot in a separate PR.

File: package.json [hunk 3]
```diff
@@ -1,8 +1,10 @@
 {
-  "name": "netlify-plugins",
+  "name": "@netlify/plugins-list",
   "private": true,
```
Comment (erezrokah) [inline package.json:3]:
```suggestion
```

**Review by erezrokah — APPROVED** [review]
Looks good, just a small change

File: package.json (hunk 3 above)
Comment (ehmicky) [reply package.json]: Good catch!

**Pull request merged**